src/main/java/org/parabot/core/ui/components/VerboseLoader.java

```diff
@@ -130,8 +133,8 @@ public void addLoginPanel() {
         usernameLabel.setAlignmentX(Box.CENTER_ALIGNMENT);
         usernameLabel.setForeground(Color.white);
 
-        final JTextField userInput = new JTextField(20);
-        final JTextField passInput = new JPasswordField(20);
+        userInput = new JTextField(new PBLocalPreferences("login.json").getSetting("login"), 20);
```

Comment:
Would it be easier if it's an accessible variable, so we don't create it twice (or even more)?

Reply:
I considered it, but I decided not to for a few reasons;

1) There wasn't a good place to create it. Neither of these two files store this kind of information, and I couldn't reason a suitable class in common
2) Once the client is loaded, you won't be needing to grab the username from the file anyways
3) This method is more concise, as defining it as a public variable would require 3 lines of code instead of 2 anyways

If you want me to put it in one of the classes regardless, I'll do that. It just didn't sit well imo.